.github/ISSUE_TEMPLATE/bug_report_bot.yml

```diff
@@ -0,0 +1,59 @@
+name: "Bug Report: Discord Bot"
+description: File a bug report for the modmail bot.
+labels: ["a: bot", "s: needs triage", "t: bug"]
+assignees: onerandomusername
+body:
+- type: input
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    placeholder: I have searched the existing issues.
+  validations:
+    required: true
```

Comment:
This needs a different input type. It should either be a search-for-issues box on the form or a checkbox, and right now it's a text input that gets pasted into the output